test/commons/aria/get-owner-virtual.js

```diff
@@ -0,0 +1,80 @@
+describe('aria.getOwnerVirtual', function() {
+	'use strict';
+
+	var fixture = document.getElementById('fixture');
+	var queryFixture = axe.testUtils.queryFixture;
+	var shadowSupported = axe.testUtils.shadowSupport.v1;
+	var aria = axe.commons.aria;
+
+	afterEach(function() {
+		fixture.innerHTML = '';
+		axe._tree = undefined;
+	});
+
+	it('returns owner element - `div` which is a direct parent (not `aria-owns`)', function() {
+		var vNode = queryFixture(
+			'<div><p role="listitem" id="target">Nothing here.</p></div>'
+		);
+
+		var owner = aria.getOwnerVirtual(vNode);
+		assert.equal(owner.nodeName.toUpperCase(), 'DIV');
```

Comment:
There can be any number of divs on this page. This test doesn't prove what you're trying to prove.